pack/ms.json

```
@@ -1,4 +1,104 @@
 [
+  {
+    "base_link": 0,
+    "code": "33001",
+    "deck_limit": 1,
+    "faction_code": "anarch",
+    "flavor": "Waiting is useless. The crisis is here; pick a side.",
+    "illustrator": "Benjamin Giletti",
+    "influence_limit": 15,
+    "keywords": "Cyborg",
+    "memory_cost": 4,
+    "minimum_deck_size": 45,
+    "pack_code": "ms",
+    "position": 1,
+    "quantity": 1,
+    "side_code": "runner",
+    "stripped_text": "The first time each turn you suffer core damage, you may draw 1 card and sabotage 2. (The Corp trashes 2 cards of their choice from HQ and/or the top of R&D.)",
+    "stripped_title": "Esa Afontov: Eco-Insurrectionist",
+    "text": "The first time each turn you suffer core damage, you may draw 1 card and sabotage 2. <em>(The Corp trashes 2 cards of their choice from HQ and/or the top of R&D.)</em>",
+    "title": "Esâ Afontov: Eco-Insurrectionist",
+    "type_code": "identity",
+    "uniqueness": false
+  },
+  {
+    "code": "33002",
+    "cost": 3,
+    "deck_limit": 3,
+    "faction_code": "anarch",
+    "faction_cost": 4,
+    "flavor": "We're all alike down here Androids are friends we salute Give us all a f***ing break Ol' Jack is the s*** on my boot.",
```

```suggestion
    "flavor": "We're all alike down here\nAndroids are friends we salute\nGive us all a f***ing break\nOl' Jack is the s*** on my boot.",
```
I think we tend to put newlines in the flavour text.